### ISSUE TITLE
Immortal Guardians Yogg-Saron

### ISSUE THREAD

**Issue by Chryslol** [SunwellTracker/issues#3014]
Decription: There is currently a **major** issue with how pathing works of these mobs. They randomly ignore taunt-effects and stand still doing nothing. The mobs also randomly evade 

How it works: When Immortal Guardians get taunted, they randomly ignore taunt-effects and stand still doing nothing. This becomes an issue on Yogg+0 light where they need to be taunted out to not heal other Immortal Guardians / Yogg-Saron. It is very obvious there's something going on. 
**Immortal Guardians also randomly evade.** 

How it should work: Immortal Guardians should follow their current target and run to it instantly if it's not casting. 

Source (you should point out proofs of your report, please give us some source): 
In this clip, Circle randomly stops at the exact spot / distance where Hunter taunts from. Immortal Guardian does **not** cast but it still doesn't follow hunter to it's location. 
https://clips.twitch.tv/PunchyPeppyFerretTBCheesePull 

In this clip, Triangle is taking ages to react to taunt- it casts, but after taunt it takes the mob 1-3 seconds to react to what happened to it. It moves out after a while, since Shadow Beacon is 10 seconds debuff, it's important that mobs react to what's happening to them.
https://www.twitch.tv/videos/578477713?t=02h37m00s

In this clip, watch **Star** closely- it is taunted by hunter- and it randomly stops at the same location where he taunts from (you can see him standing on a spot, then taunts then runs back).
https://clips.twitch.tv/PuzzledBrightHareLitFam

In this clip, Circle stops again at the same spot as hunter taunts. 
https://clips.twitch.tv/MiniatureMotionlessBeefSpicyBoy

In this clip, Circle casts, then stops casting, runs to hunter then randomly start running back and forth eventho it has Distracting Shot debuff(it also changes target with debuff up).

In this clip, Circle first casts, then gets interrupted, takes the mob 2 seconds to move. 
https://clips.twitch.tv/ExuberantDistinctLyrebirdBrainSlug

In this clip, there's a mob evading when it's away from group, casting it's Drain Life then randomly evades. Stands still for 10 seconds, then gets into fight again (full HP but small, minor issue but still annoying).
https://clips.twitch.tv/FunnyTolerantBulgogiBudStar

**Comment by deathstreal** [comment]
This does not only happen on Immortal Guardians, it happens on every mob quite often.

**Comment by Chryslol** [comment]
@deathstreal yeah, this is probably a server-issue. It probably has to do with server-lag aswell but devs need to look into it.
